modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountRoleResourceTest.java

```diff
@@ -949,6 +949,8 @@ private UserAccount _randomUserAccount() throws Exception {
 
 						return calendar.getTime();
 					});
+
+				setStatus(Status.ACTIVE);
```

Comment:
Is not possible to have `status = Status.ACTIVE` ?

Reply:
It is, i changed it now